src/components/MaterialUI/Inputs/Checkbox.js

```diff
@@ -117,7 +118,7 @@ const Checkbox = ({ checkboxProps }) => {
 	const checkBoxMui = (
 		<CheckboxMUI
 			checked={value}
-			onChange={handleChange}
+			onChange={!readOnly ? handleChange : null}
```

Comment:
Does CheckboxMUI has disabled property? Beacuse it should be disabled for Readers (as we do for Inputs

Reply:
@InnaBoitsun 
We need to align readOnly and disabled props for most of input components with style guide create by Alain. Since currently in most of them we have just 1 but, need both.

We should do it as part of separate US and before it I'm suggesting to not care about it too much.

Reply:
ok, but it is not hard to add disabled property here and use default MUI view for this. But as you wish.